src/_locales/en/messages.json

```diff
@@ -8,7 +8,7 @@
         "description": "text for browser action"
     },
     "storageText": {
-        "message": "Can store data upto 5 MB",
+        "message": "Can store data up to 5 MB",
```

Comment:
Thanks for noticing this @spoji! ⭐️

Reply:
Just saw this typo when @tsl143 presented the extension on Mozilla addons-demo room! 

Nice tool!

Reply:
Hi Spoji its landed in latest version now. Thank you.